src/components/BlockchainsStack/BlockchainsStack.module.scss

```diff
@@ -142,3 +149,15 @@
   font-size: var(--text-md);
   font-weight: var(--medium);
 }
+
+.tooltip {
+  max-width: 305px;
+  white-space: normal;
+  overflow-wrap: break-word;
+  border-radius: var(--space-sm-fixed);
+  padding: var(--space-2lg-fixed);
```

Comment:
If we want to keep the tooltip, it would be better to decrease the padding to match the size of our other tooltips

Reply:
nice catch